src/ray/raylet/worker_pool.cc

```diff
@@ -416,7 +428,11 @@ Process WorkerPool::StartWorkerProcess(
                                worker_type);
   state.starting_worker_processes.emplace(
       worker_startup_token_counter_,
-      StartingWorkerProcessInfo{workers_to_start, workers_to_start, worker_type, proc});
+      StartingWorkerProcessInfo{workers_to_start, workers_to_start, worker_type, proc,
+                                runtime_env_info});
+  runtime_env_manager_.AddURIReference(
+      kWorkerSetupTokenPrefix + std::to_string(worker_startup_token_counter_),
```

Comment:
Can we also append the task or actor id? That'll help us debugging issues

Reply:
We can't map one worker process to a task because the worker process can be cached and reused for normal task. Maybe the only thing we can do is appending actor id for dedicated workers? We can guarantee that a dedicated worker occupy a worker process now.

Reply:
Hmm in this case, I think it is fine!